setup.cfg

```diff
@@ -34,3 +34,22 @@ ignore_missing_imports = True
 [flake8]
 max-line-length = 200
 exclude = venv/*,var/*
+
```

Comment:
Thanks for adding the error code comments. Can we organize the comments a bit better? Maybe something like this:

```
# Ignore completely:
# <flake8 code> - <description>
# <flake8 code> - <description>
#
# Ignore on a per file basis:
# <flake8 code> - <description>
# <flake8 code> - <description>
```

Reply:
done